packages/p2p/src/stores/order-store.js

```diff
@@ -145,19 +144,17 @@ export default class OrderStore {
                         response?.error.code === api_error_codes.INVALID_VERIFICATION_TOKEN ||
                         response?.error.code === api_error_codes.EXCESSIVE_VERIFICATION_REQUESTS
                     ) {
-                        clearTimeout(wait);
                         if (this.is_email_verification_modal_open) {
                             this.setIsEmailVerificationModalOpen(false);
                         }
                         if (this.is_email_link_verified_modal_open) {
                             this.setIsEmailLinkVerifiedModalOpen(false);
                         }
-                        this.setVerificationLinkErrorMessage(response.error.message);
-                        const wait = setTimeout(() => this.setIsInvalidVerificationLinkModalOpen(true), 230);
+                        this.root_store.general_store.showModal({
+                            key: 'InvalidVerificationLinkModal',
+                            props: { error_message: response.error.message, order_id: id },
```

Comment:
```suggestion
                            props: { error_message: response?.error.message, order_id: id },
```

Reply:
we are already checking response on line 138